docs/reference/api/docker_remote_api_v1.25.md

```diff
@@ -3224,6 +3393,38 @@ Instruct the driver to remove the volume (`name`).
 -   **409** - volume is in use and cannot be removed
 -   **500** - server error
 
+### Prune unused volumes
+
+`POST /volumes/prune`
+
+Delete unused volumes
+
+**Example request**:
+
+    POST /volumes/prune HTTP/1.1
+    Content-Type: application/json
+
+    {
+    }
+
+**Example response**:
+
+    HTTP/1.1 200 OK
+    Content-Type: application/json
+
+	{
+	    "VolumesDeleted": [
+	        "my-volume"
```

Comment:
Oh, I see.

They do act on different objects, so it makes sense to me that they would differ in output.